istantest.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+NPM=$(which npm)
+ROOTPATH=$(pwd)
+TESTBED='testland'
+TESTBEDPATH=$(echo $ROOTPATH/$TESTBED)
+sudo rm -rf $TESTBED
+echo "Instrumenting.."
+istanbul instrument . -o $TESTBED
```

Comment:
Maybe I'm missing something, but why do you pre-instrument the code like this? Doesn't [`istanbul cover`](https://github.com/gotwarlost/istanbul#the-cover-command) instrument the code? I don't think we want the code doubly instrumented...

I only ask because this is confusing coveralls.